provider-messaging/src/components/organisms/c-edit-profile.jsx

```diff
@@ -26,7 +52,7 @@ const CEditProfile = ({ profile = usersData[0], opened, setOpenModal }) => {
           <div className="o-aspect-ratio w-40 mb-7 mx-auto">
             <img
               className="o-aspect-ratio__content rounded-2xl object-cover"
-              src={profile.avatar}
+              src={profile.avatar || "https://images.unsplash.com/photo-1611695434398-4f4b330623e6?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=435&q=80"}
```

Comment:
@dimdejesus we should move away from externally hosted resources including CDNs for InfoSec compliance reasons. (since we are a health tech)